### DISCUSSION TITLE
Error saving changes in settings

### DISCUSSION THREAD

**Discussion by mrfabe** [sct/overseerr#2137] [Help] [answered]
I was playing around the Enable CSRF Protection option and enabled it, saved it, and restarted my Overseerr session.  Afterwards, I tried to uncheck the option and save the settings but receive a "Something went wrong while saving settings" message.  I am also unable to make any other changes in the other setting options without receiving the same error.  Do I need to do a complete wipe and reinstall to resolve this issue?

**Comment by TheCatLady** [comment, marked as answer]
https://docs.overseerr.dev/using-overseerr/settings#enable-csrf-protection

**Reply by mrfabe** [reply]
Thanks... I had to try it.

I reset and reconfigured Overseerr... simple, easy, and fast

**Reply by TheCatLady** [reply]
But you didn't need to reset. 😅  The solution is in the linked docs.

**Comment by mrfabe** [comment]
I saw that in the config file but wasn't 100% sure that the save issue was because of CSRF change.
Thank you for your input and help!
